### PR TITLE
Add Emacs README 

### PR DESCRIPTION
**Pull request by ztlevi**
I cannot believe Emacs is not on your list😂.. But anyway, I will do it. Hopefully, this can help someone.

### CODE REVIEW

**Comment by sb2nov** [comment]
Thanks. you might have to add an entry in https://github.com/sb2nov/mac-setup/blob/master/SUMMARY.md to generate this.

File: Emacs/README.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+12. <code>--without-libxml2</code>: Build without libxml2 support<br>
+13. <code>--without-modules</code>: Build without dynamic modules support<br>
+14. <code>--without-multicolor-fonts</code>: Build without a patch that enables multicolor font support<br>
+15. <code>--without-spacemacs-icon</code>: Build without Spacemacs icon by Nasser Alshammari<br>
+16. <code>--HEAD</code>: Install HEAD version<br>
+</details>
+
+> Note: You might want to install [exec-path-from-shell](https://github.com/purcell/exec-path-from-shell) if you are using Emacs plus. It takes care of your environment variables.
+
+> Note: to have the title bar match your theme background color, consider using instead:
```
Comment (Kyslik) [inline Emacs/README.md]: Missing bold on note as well.

File: Emacs/README.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+10. <code>--without-imagemagick@6</code>: Build without imagemagick@6 support<br>
+11. <code>--without-librsvg</code>: Build without librsvg support<br>
+12. <code>--without-libxml2</code>: Build without libxml2 support<br>
+13. <code>--without-modules</code>: Build without dynamic modules support<br>
+14. <code>--without-multicolor-fonts</code>: Build without a patch that enables multicolor font support<br>
+15. <code>--without-spacemacs-icon</code>: Build without Spacemacs icon by Nasser Alshammari<br>
+16. <code>--HEAD</code>: Install HEAD version<br>
+</details>
+
+> Note: You might want to install [exec-path-from-shell](https://github.com/purcell/exec-path-from-shell) if you are using Emacs plus. It takes care of your environment variables.
```
Comment (Kyslik) [inline Emacs/README.md]: Make "Note" bold

```md
> **Note**: ...
```

File: Emacs/README.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+   Or you can set the `dotspacemacs-elpa-https` to `nil` in your dotfile to
+   remove the need to start Emacs with `--insecure` argument. You may wish to
+   clear out your `.emacs.d/elpa` directory before doing this, so that any
+   corrupted packages you may have downloaded will be re-installed.
+
+5. Restart Emacs to complete the installation.
+
+## Purcell's Emacs configuration
+
+This is [Purcell's](https://github.com/purcell/emacs.d) emacs configuration tree, continually used and tweaked since 2000, and it may be a good starting point for other Emacs users, especially those who are web developers. These days it's somewhat geared towards OS X, but it is known to also work on Linux and Windows.
```
Comment (Kyslik) [inline Emacs/README.md]: macOS instead of OS X

**Comment by ztlevi** [comment]
All fixed. Happy new year, guys!

**Pull request merged**